evap/evaluation/models_logging.py

```diff
@@ -113,7 +115,7 @@ def message(self):
             message = _("A {cls} was deleted.")
 
         return message.format(
-            cls=self.content_type.model_class()._meta.verbose_name_raw,
+            cls=capitalize_first(self.content_type.model_class()._meta.verbose_name),
```

Comment:
why did we use _raw here before (_raw should be the untranslated version - is it possible that translation is in some cases not available, which will break this code?)

Reply:
@felixrindt did you have any specific reasons for that?

Reply:
No, I suppose it works without raw as well 😅

Reply:
i think so too. we shouldn't have any user-facing strings that aren't translated so everything should be fine.